webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableActions.js

```diff
@@ -0,0 +1,43 @@
+import {
+  TABLE_REQUEST,
+  TABLE_SUCCESS,
+  TABLE_FAILURE,
+  INIT_TABLE,
+} from './ModelsTableConsts';
+import { ajaxRequestAction } from '../../redux/actions/common';
+
+export const updateSearch = (current) => {
+  const init = {
+    page: current.params && current.params.page,
+    perPage: current.params && current.params.per_page,
+    order: current.params && current.params.order,
+    searchQuery: current.params && current.params.search,
+    results: current.table,
+  };
+
+  const payload = Object.keys(init)
```

Comment:
IMO it feels hacky and it's difficult to follow and understand how the payload ends up.
Would it be easier if the action receive a destructed object?
```js
export const updateSearch = ({ page, perPage, order, searchQuery, table }) => { ... };
```